draft-ietf-openpgp-pqc.md

```diff
@@ -668,7 +668,9 @@ Implementations MUST NOT use the obsolete Symmetrically Encrypted Data packet (t
 
 ### Key Material Packets {#mlkem-ecc-key}
 
-The composite ML-KEM + ECDH schemes MUST be used only with v6 keys, as defined in [RFC9580], or newer versions defined by updates of that document.
+The composite ML-KEM-768 + X25519 (algorithm ID 35) MUST be used only with v4 or v6 keys, as defined in [RFC9580], or newer versions defined by updates of that document.
+
+The composite ML-KEM-1024 + X448 (algorithm ID 36) MUST be used only with v6 keys, as defined in [RFC9580], or newer versions defined by updates of that document
```

Comment:
This seems clunky, but i think the text is correct.

If you want to be friendlier, you could also put a note at the top of the draft (in the introduction?) that says something like "these asymmetric algorithms should be used only in v6 TPKs and TSKs, with the single exception of algorithm ID 35.  Algo 35 is also permitted for use in v4 TPKs and TSKs for the purpose of defending OpenPGP messages encrypted to existing certificates against store-now, decrypt-later attacks that use quantum computing machinery."

Reply:
I pushed a version of this suggestion. We'll discuss it tomorrow morning